### ISSUE TITLE
NullReferenceException when parsing Global.yml

### ISSUE THREAD

**Issue by andy0x10** [Jan0660/DocFxMarkdownGen#9]
v0.3.0
Hit the following exception:
```
[INFO] DocFxMarkdownGen v0.3.0 running...
[INFO] Read all YAML in 12ms.
[INFO] Generating and writing markdown...
Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at Program.<>c__DisplayClass0_0.<<<Main>$>b__15>d.MoveNext() in /home/runner/work/DocFxMarkdownGen/DocFxMarkdownGen/Program.cs:line 210
--- End of stack trace from previous location ---
   at System.Threading.Tasks.Parallel.<>c__50`1.<<ForEachAsync>b__50_0>d.MoveNext()
--- End of stack trace from previous location ---
   at Program.<Main>$(String[] args) in /home/runner/work/DocFxMarkdownGen/DocFxMarkdownGen/Program.cs:line 208
   at Program.<Main>(String[] args)
```

when parsing the Docfx generated Global.yml:

```
### YamlMime:ManagedReference
items:
- uid: Global
  id: Global
  children:
  - Global.UpdateTest
  - Global.VertexGrid
  langs:
  - csharp
  - vb
  name: ''
  nameWithType: ''
  fullName: ''
  type: Namespace
  assemblies:
  - TextReveal.Runtime
references:
- uid: Global.VertexGrid
  commentId: T:Global.VertexGrid
  name: VertexGrid
  nameWithType: VertexGrid
  fullName: VertexGrid
- uid: Global.UpdateTest
  commentId: T:Global.UpdateTest
  name: UpdateTest
  nameWithType: UpdateTest
  fullName: UpdateTest

```

**Comment by Jan0660** [comment]
Couldn't reproduce but tried to fix this in v0.3.1.

**Comment by andy0x10** [comment]
Yes that's fixed it 👍
